### ISSUE TITLE
The generated page Units and Constants is broken

### ISSUE THREAD

**Issue by lesquoyb** [gama-platform/gama-platform.github.io#118]
**Describe the bug**
There are two problems:

  * The table of contents do not work on the [units and constants](https://gama-platform.org/wiki/UnitsAndConstantsPDF#table-of-contents) page. 
It shows `<wiki:toc max_depth="3" />` instead of the actual table of contents.
This page being generated automatically it cannot be fixed manually.
  * all the constants are displayed with `\#` before there name instead of just `#` or juste the name, and they are followed by `value=` then the name again, which also looks like a problem in the generation process

**Comment by lesquoyb** [comment]
Update: the broken table of contents and the `\#` before constants' names are actually not on the UnitsAndConstants page, but on the UnitsAndConstants**PDF** page.
I guess that this page is a temporary page used to generate the pdf of the documentation. 
So, the problem is that this page is the first result in the search engine whilst it shouldn't be accessible at all if it's only an intermediary file.
Also, the problem with the formatting of the constants' descriptions (`value=` before the value, and `comment: ` before comments) 
remains present in the *good* [UnitsAndConstants](https://gama-platform.org/wiki/next/UnitsAndConstants) page and should be fixed.

**Comment by RoiArthurB** [comment]
You're right, this page shouldn't be on the website. I pushed a fix to prevent new people to arrive at the shard of the PDF version of the documentation :) 

For the pseudo format "<code> value: comment:" it's a design choice to keep everything ordered and on a single line. It's not a problem from my point of view but please don't hesitate to make it as a table on the page generation if you feel more like it.

**Comment by lesquoyb** [comment]
I think we should keep coherent with the design of other pages that are formatted as:
`name` (`type`): `comment`
e.g.:
![image](https://user-images.githubusercontent.com/6374469/187151227-7b844fcf-abd7-4e50-9fa1-406071f22cf3.png)

Which in our case would give:
`name` (`value`): `comment`
And it is actually more compact

Also, I don't know if it's something lost in the generation of the documentation or just the formatting, but the value of string constants are not in between quotes.

**Comment by RoiArthurB** [comment]
I would agree that it'll be more coherent over the whole website, but we may lose comprehension in the information. 

This page makes a list of each constant (concept starting with the key-character `#` in gaml) over everything existing in gaml (going from colors to light's position for instance). So the `value` can be very different between things (float, int, string, else; cf screenshot below) and don't always fit in parentheses as a type would from my point of view.

![image](https://user-images.githubusercontent.com/16764085/187153839-44afba25-2e5b-45b1-89ea-e2593cc43f5f.png)

---

Long story short, I prefer this format for this page keeping heterogeneous data still easy to read in a short one-liner :)
